flexible/java-11/cloudstorage/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,10 @@
+runtime: java
+env: flex
+runtime_config:
+  operating_system: ubuntu18
```

Comment:
[optional] This is a code sample, so readability and demonstrative matters a lot. Customers may wonder why ubuntu18 maps to Java 11. You could add a short comment here, explaining that we are pivoting to OS' perspective. Java 11 is the latest java runtime that is supported in ubuntu 18.

Reply:
That's a different behavior though. So, I guess it depends on the goal of this sample. Do you want to:
1) Demonstrate how default java version is selected when operating_system is set to ubuntu18. Or,
2) Demonstrate how to use (pin to) java 11. 

I don't see a clear winner here. Please feel free to pick whichever makes more sense to you.